File: 3110-ScoreOfAString.java
// https://leetcode.com/problems/score-of-a-string/

class Solution {
    public int scoreOfString(String s) {
        int score = 0, length = s.length();
        for (int i = 1; i < length; i++) {
            score += Math.abs(s.charAt(i) - s.charAt(i - 1));
        }
        return score;
    }
}

File: 3110-score_of_a_string.py
# https://leetcode.com/problems/score-of-a-string/

class Solution:
    def scoreOfString(self, s: str) -> int:
        score, length = 0, len(s)
        for i in range(1, length):
            score += abs(ord(s[i]) - ord(s[i - 1]))
        return score
